.circleci/config.yml

```diff
@@ -4,6 +4,13 @@ jobs:
   build:
     docker:
       - image: circleci/python:3.9
+        environment:
+          DATABASE_URL: postgresql://postgres@localhost/circle_test?sslmode=disable
+      - image: circleci/postgres:13.1
+        environment:
+          POSTGRES_DB: circle_test
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
```

Comment:
leaving the password in this file is okay right? :0

Reply:
good question! i think this should be ok - the database is only temporary, and will only be filled with test data to run our tests anyway 😄 ([source](https://discuss.circleci.com/t/how-to-securely-set-environment-variables-for-postgres-image/36667))